fragment_internal_test.go

```diff
@@ -2572,3 +2572,29 @@ func TestFragmentRowIterator(t *testing.T) {
 		}
 	})
 }
+
+func TestUnionInPlaceMapped(t *testing.T) {
+	f := mustOpenFragment("i", "f", "v", 0, CacheTypeNone)
+	defer f.Clean(t)
+	r0 := rand.New(rand.NewSource(2))
+	r1 := rand.New(rand.NewSource(1))
+	data0 := randPositions(1000000, r0)
+	setBM := roaring.NewBitmap()
+	setBM.OpWriter = nil
+	setBM.Add(data0...)
+	unprotectedWriteToFragment(f, setBM)
+	data1 := randPositions(1000000, r1)
+	setBM2 := roaring.NewBitmap()
+	setBM2.OpWriter = nil
+	setBM2.Add(data1...)
+
+	f.storage.UnionInPlace(setBM2)
+}
```

Comment:
It seems to me like we should check the results in some way? But I'm not sure how, since we don't have a direct check for mapping, and the original wasn't mapped.